src/utils.ts

```diff
@@ -7,22 +8,11 @@ import * as React from 'react'
  */
 export function equals<T>(a: T, b: T): boolean {
   if (isImmutable(a) && isImmutable(b)) {
-    return a.is(b)
```

Comment:
I got an error from here, because is` method is not an member of List class.

Reply:
Question - if we just changed this line to `.equals` instead of `.is`, would that fix your issue?

Reply:
Looks yes. If so, we don't have to have `immutable-is` anymore and it would be better.

Reply:
Let's do that, and keep the test you wrote.